code/_onclick/other_mobs.dm

```diff
@@ -7,6 +7,21 @@
 /mob/living/carbon/human/UnarmedAttack(atom/A, proximity)
 
 	if(!has_active_hand()) //can't attack without a hand.
+		if(a_intent == INTENT_HELP && src == A)
+			check_self_for_injuries()
+			return
+
+		var/obj/item/bodypart/check_arm = get_active_hand()
+		if(!check_arm)
+			to_chat(src, "<span class='notice'>You look at your arm and sigh.</span>")
+			return
+
+		for(var/i in check_arm.wounds)
+			var/datum/wound/W = i
+			if(W.disabling)
+				to_chat(src, "<span class='warning'>The [lowertext(W.name)] in your [check_arm.name] is preventing you from using it! Get it fixed, or at least splinted!</span>")
+				return

```

Comment:
Add a `TRAIT_LIMB_DISABLED` trait and then do a `HAS_TRAIT_FROM(src, TRAIT_LIMB_DISABLED, check_arm.type)`

Alternatively use a check for which of the hands is the active one and check for either `TRAIT_PARALYSIS_R_ARM` or `TRAIT_PARALYSIS_L_ARM`, but it would be better if these two tied into a more global trait like the one mentioned before.